v2/codetags/parser.go

```diff
@@ -371,27 +363,11 @@ parseLoop:
 			}
 		case stMaybeComment:
 			switch {
-			case s.skipWhitespace():
-			case r == '/':
-				s.next()
-				incomplete = true
-				st = stTrailingSlash
-			default:
-				break parseLoop
-			}
-		case stTrailingSlash:
-			switch {
-			case r == '/':
-				s.next()
-				incomplete = false
-				st = stTrailingComment
+			case r == '/' && s.peekN(1) == '/':
```

Comment:
We could switch tag line-comments to use `#` instead - better or worse?

Reply:
I prefer it.  I find the use of `//` as a comment tag inside other `//` comments :exploding_head: 

We did support `//` before this change, but only for a few months.  Are we okay to break?  If so I'm happy to make the change.

Reply:
I think a break is OK